handbook/company/pricing-features-table.yml

```diff
@@ -613,15 +613,16 @@
   tier: Premium
   jamfProHasFeature: yes
   jamfProtectHasFeature: yes
-- industryName: Interactive MDM migration #   « end-user initiated MDM migration, with interactive UI
+- industryName: MDM migration #   « end-user initiated MDM migration, with interactive UI
   tier: Premium
   jamfProHasFeature: yes
   jamfProtectHasFeature: no
   documentationUrl: https://fleetdm.com/docs/using-fleet/mdm-migration-guide#migrate-automatically-enrolled-dep-hosts
   usualDepartment: IT
   productCategories: [Device management]
   pricingTableCategories: [Device management]
-- industryName: Remotely enforce OS settings
+- industryName: Enforce OS settings
```

Comment:
Consolidated `Remotely enforce OS settings` and `Low-level MDM commands for macOS, iOS/iPadOS*, and Windows (e.g., remote restart)` and renamed to **“Enforce OS settings”**
These are all separate for the product, but for macAdmins, they are sub-features of enforcing OS settings. Added the specific operating systems to the description/tooltip.